src/sentry/utils/query.py

```diff
@@ -224,10 +224,10 @@ def __iter__(self):
         ]
         pbar = progressbar.ProgressBar(
             widgets=widgets,
-            max_value=self.count,
+            maxval=self.count,
             # The default update interval is every 0.1s,
             # which for large migrations would easily logspam GoCD.
-            min_poll_interval=10,
+            poll=10,
```

Comment:
Should these changes be included?

Reply:
Yes, because our current version of the progress bar doesn't work without them.  I don't understand, but I presume a recent package upgrade changed the versioning?  Anyways, it seems like these existing kwds just changed names for some reason and I can validate that.

Reply:
Also they don't run in tests anywhere -- it only comes up if you try to migrate the database through a migration that depends on this.

Reply:
Wait, oh, maybe I got it backwards?  Maybe my versioning was wrong.  Reverting.